mods/ra/maps/soviet-05/main.lua

```diff
@@ -0,0 +1,222 @@
+										mcvGG.Deploy()
+										IslandTroops()
+										Trigger.AfterDelay(DateTime.Seconds(7), function()
+											BuildBase()
+										end)
+									end
+									if not mcvtransport.IsDead then
+										mcvtransport.Move(ReinfNorthPoint.Location)
+										Trigger.AfterDelay(DateTime.Seconds(7), function()
+											if not mcvtransport.IsDead then
+												mcvtransport.Destroy()
+											end
+										end)
+									end
+								end)
+							end
+						end)
+					end
+				end)
+			end
+		end)
+	end
+end
+
+Tick = function()
+	if Greece.HasNoRequiredUnits() and GoodGuy.HasNoRequiredUnits() then
+		player.MarkCompletedObjective(KillAll)
+	end
+
+	if player.HasNoRequiredUnits() then
+		player.MarkFailedObjective(CaptureObjective)
+		player.MarkFailedObjective(KillAll)
+		GoodGuy.MarkCompletedObjective(BeatUSSR)
+	end
+
+	if Radar.IsDead then
+		player.MarkFailedObjective(CaptureObjective)
+	end
+
+	if Radar.Owner == player and not Radstop then
+		if not ExpansionCheck then
+			Expand()
+			ExpansionCheck = true
+		end
+
+		player.MarkCompletedObjective(CaptureObjective)
+
+		Reinforcements.Reinforce(Greece, ArmorReinfGreece, AlliedCrossroadsToRadarPath , 0, function(soldier)
+			soldier.Hunt()
+		end)
+
+		Radstop = true
+	end
+
+	if not baseEstablished and CheckForBase() then
+		baseEstablished = true
+		Para()
+	end
+
+	if not SPenEstablished and CheckForSPen() then
+		SPenEstablished = true
+
+		local units = Reinforcements.ReinforceWithTransport(Greece, "lst", arty1reinf, SouthReinfPath, { ReinfEastPoint.Location })[2]
+		Utils.Do(units, function(unit) IdleHunt(unit) end)
+		if not ExpansionCheck then
+			Expand()
+			ExpansionCheck = true
+		end
+	end 
+
+	if not RCheck then
+		RCheck = true
+		if Map.Difficulty == "Easy" and ReinfCheck then  
+			Trigger.AfterDelay(DateTime.Minutes(6), ReinfArmor) 
+		elseif Map.Difficulty == "Medium" then  
+			Trigger.AfterDelay(DateTime.Minutes(4), ReinfArmor) 
+		else  
+			Trigger.AfterDelay(DateTime.Minutes(3), ReinfArmor) 
+		end
+	end
+end
+ 
+WorldLoaded = function()
+	player = Player.GetPlayer("USSR")
+	GoodGuy = Player.GetPlayer("GoodGuy")
+	Greece = Player.GetPlayer("Greece")
+
+	RunInitialActivities()
+
+	CaptureObjective = player.AddPrimaryObjective("Capture the Radar.")
+	KillAll = player.AddPrimaryObjective("Defeat the Allied Forces.")
+	BeatUSSR = GoodGuy.AddPrimaryObjective("Defeat the USSR Forces.")
+
+	Trigger.OnDamaged(mcvGG, Expand)
+	Trigger.OnDamaged(mcvtransport, Expand)
+
+	Trigger.OnEnteredProximityTrigger(SovietExpansionPoint.CenterPosition, WRange.New(4 * 1024), function(unit, id)
+		if unit.Owner == player and Radar.Owner == player then
+			Trigger.RemoveProximityTrigger(id)
+
+			Para2()
+			ProduceInfantryGG()
+			ProduceTanksGG()
+
+			local units = Reinforcements.ReinforceWithTransport(player, "lst", {"mcv", "3tnk", "3tnk", "e1", "e1"}, { ReinfSouthPoint.Location, SovietExpansionUnloadPoint.Location }, { ReinfSouthPoint.Location })[2]
```

Comment:
Minor nit: Spaces before/after the brackets please.
`{ "mcv", "3tnk", "3tnk", "e1", "e1" },`